Standards/scs-XXXX-v1-security-of-iaas-service-software.md

```diff
@@ -0,0 +1,108 @@
+Security Updates and Bug fixes are also provided by OpenStack for a few older releases with the state `maintained`.
+Additionally the [SCS reference implementation](https://github.com/SovereignCloudStack/release-notes/blob/main/Release7.md) is integrating OpenStack releases after half a year - so about the time when a new release is cut by OpenStack.
+Considering a CSP that wants to use only SLURP releases and waiting for the reference implementation will already be over a year or 2 releases behind, which cannot be considered as using the current version of IaaS Layer Software.
+Thus this option can be discarded.
+
+#### Allow only maintained versions of Software
+
+While following closely to the newest releases could be advised, there are several downsides to requiring this workflow, even if it would be only for SLURP releases.
+Following the SCS reference implementation for example would also lead into being a little bit behind the newest release.
+But this is not as bad as it may seem to be, because security related fixes and bug fixes are backported to older but still `maintained` releases.
+All releases that are still maintained can be looked up at the releases page from OpenStack[^1].
+
+Allowing maintained versions would give CSPs a little bit more time to update and test their environments, while still receiving relevant security updates and bug fixes.
+Also CSPs that want to become SCS-compliant may not have the burden to upgrade their deployments, but can test before an upgrade, where they need to put in additional work to become SCS-compliant.
+
+One problem is, that there might be new features implemented in the newest versions of the software, which are desired by other standards to be SCS-compliant.
+In that case allowing all maintained versions would lead to a two-year timespan customers would need to wait for before such a feature becomes available in all SCS-compliant deployments.
+In case of security relevant features this is not advisable.
+
+#### Standards implicitly define the minimum versions of Software
+
+Instead of requiring a defined minimum software version, it could be derived from the standards.
+Because: Whenever there is a new wanted behavior a standard should be created and a resonable timeframe given to CSPs to adopt a software version that can fulfill the new standard.
+Through the combination of all standards that are in place, the minimum version for the IaaS service software is implicitly given.
+
+This would avoid to have conflicting versions of software in terms of feature parity, while also allowing older software.
+Using this approach requires an additional advise to CSPs to update or implement patches for security issues.
+
+#### Advise CSPs to integrate software updates
+
+As long as maintained versions of software are used, updates with security patches are available and only need to be integrated.
+This can and should be done in a reasonable short timeframe.
```

Comment:
What is a reasonable time? IMO, standard should be explicit here in order to guide CSPs.

How does customers know, whether or not a specific bug is fixed? Do we "force" CSPs to document software version/patch version somewhere?